lib/fs.js

```diff
@@ -1193,7 +1208,10 @@ function toUnixTimestamp(time) {
     // convert to 123.456 UNIX timestamp
     return time.getTime() / 1000;
   }
-  throw new Error('Cannot parse time: ' + time);
+  throw new errors.Error('ERR_INVALID_ARG_TYPE',
```

Comment:
1. These a disparity with the docs (but out of scope for this PR)
2. AFAICT `date string` will not parse

tl;dr 3rd arg should be `['Date', 'time in seconds']`

Reply:
2.You are right I will go with ['Date', 'time in seconds']
The date string was (inaccurate) for a string timestamp arg like '1504020561954'

Reply:
I updated it to `['Date', 'time in seconds']`